sys/php/www.conf

```diff
@@ -404,25 +433,12 @@ slowlog = /var/log/php-fpm/www-slow.log
 ; instead.
 
 ; Note: path INI options can be relative and will be expanded with the prefix
-; (pool, global or @prefix@)
+; (pool, global or /usr/local)
 
 ; Default Value: nothing is defined by default except the values in php.ini and
 ;                specified at startup with the -d argument
 ;php_admin_value[sendmail_path] = /usr/sbin/sendmail -t -i -f www@my.domain.com
 ;php_flag[display_errors] = off
-php_admin_value[error_log] = /var/log/php-fpm/www-error.log
-php_admin_flag[log_errors] = on
-;php_admin_value[memory_limit] = 128M
-
-; Set the following data paths to directories owned by the FPM process user.
-;
-; Do not change the ownership of existing system directories, if the process
-; user does not have write permission, create dedicated directories for this
-; purpose.
-;
-; See warning about choosing the location of these directories on your system
-; at http://php.net/session.save-path
-php_value[session.save_handler] = files
-php_value[session.save_path]    = /var/lib/php/session
-php_value[soap.wsdl_cache_dir]  = /var/lib/php/wsdlcache
-;php_value[opcache.file_cache]  = /var/lib/php/opcache
+;php_admin_value[error_log] = /var/log/fpm-php.www.log
```

Comment:
shouldn't it be `stderr`?

Reply:
This is commented out, but if I'm not wrongly there's another config that is using the stderr file descriptor.